components/org.wso2.carbon.identity.api.server.application.management/org.wso2.carbon.identity.api.server.application.management.v1/src/main/java/org/wso2/carbon/identity/api/server/application/management/v1/impl/ApplicationsApiServiceImpl.java

```diff
@@ -187,6 +195,21 @@ public Response getInboundSAMLConfiguration(String applicationId) {
         return Response.ok(samlSp).build();
     }
 
+    @Override
+    public Response getLoginFlowResult(String operationId) {
```

Comment:
Method name is not self descriptive.
Better to include a word that gives the meaning that this is for login flow generation